ci-operator/config/openshift-kni/eco-ci-cd/openshift-kni-eco-ci-cd-main__sno-hub-ztp-stable.yaml

```diff
@@ -0,0 +1,64 @@
+base_images:
+  base:
+    name: "4.15"
+    namespace: ocp
+    tag: ansible
+  cli:
+    name: "4.15"
+    namespace: ocp
+    tag: cli
+build_root:
+  project_image:
+    dockerfile_literal: |
+      FROM registry.ci.openshift.org/ci/telco-runner
+      RUN dnf -y install rsync
+releases:
+  latest:
+    release:
+      architecture: amd64
+      channel: candidate
+      version: "4.15"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: telcov10n-sno-hub-ztp-f360
+  cluster: build05
+  cron: 1 1 1 1 *
+  steps:
+    cluster_profile: equinix-ocp-metal-qe
+    env:
+      ADDITIONAL_WORKER_ARCHITECTURE: x86_64
+      ADDITIONAL_WORKERS: "1"
+      ADDITIONAL_WORKERS_DAY2: "true"
+      AGENT_PLATFORM_TYPE: none
+      AUX_HOST: openshift-qe-metal-ci.arm.eng.rdu2.redhat.com
+      DISCONNECTED: "false"
+      EXPIRATION_TIME: +5 hours
+      MCH_NAMESPACE: open-cluster-management
+      OPERATORS: |
+        [
+          {"name": "local-storage-operator", "source": "redhat-operators", "channel": "!default", "install_namespace": "openshift-local-storage", "target_namespaces": "openshift-local-storage", "operator_group": "local-operator-group"},
+          {"name": "advanced-cluster-management", "source": "redhat-operators", "channel": "!default", "install_namespace": "open-cluster-management", "target_namespaces": "open-cluster-management", "operator_group": "acm-operator-group"},
+          {"name": "topology-aware-lifecycle-manager", "source": "redhat-operators", "channel": "!default", "install_namespace": "openshift-talm", "target_namespaces": "", "operator_group": "topology-aware-lifecycle-manager"},
+          {"name": "openshift-gitops-operator", "source": "redhat-operators", "channel": "!default", "install_namespace": "openshift-gitops-operator", "target_namespaces": "", "operator_group": "gitops-operator-group"}
+        ]
+      RESERVE_BOOTSTRAP: "false"
+      SHARED_HUB_CLUSTER_PROFILE: ztp-hub-preserved-prod-cluster_profile_dir
```

Comment:
@aleskandro this will be the folder name that both Hub and Spoke would share at AUX_HOST:/var/telco-qe-preserved/${SHARED_HUB_CLUSTER_PROFILE}